lib/components/chat_history/twitch/cheer_event.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:intl/intl.dart';
+import 'package:provider/provider.dart';
+import 'package:rtchat/models/messages/twitch/event.dart';
+import 'package:rtchat/models/style.dart';
+
+class TwitchCheerEventWidget extends StatelessWidget {
+  final TwitchCheerEventModel model;
+  final NumberFormat _formatter = NumberFormat.compactSimpleCurrency();
+
+  TwitchCheerEventWidget(this.model, {Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    var name = model.isAnonymous ? 'Anonymous' : model.giverName;
+    return Consumer<StyleModel>(builder: (context, styleModel, child) {
+      var boldStyle = Theme.of(context)
+          .textTheme
+          .bodyText2!
+          .copyWith(fontSize: styleModel.fontSize, fontWeight: FontWeight.w500);
+      var baseStyle = Theme.of(context)
+          .textTheme
+          .bodyText2!
+          .copyWith(fontSize: styleModel.fontSize);
+      return Container(
+        decoration: BoxDecoration(
+          border: Border(
+            left: BorderSide(
+              width: 4,
+              color: Theme.of(context).accentColor,
+            ),
+          ),
+        ),
+        child: Padding(
+          padding: const EdgeInsets.fromLTRB(1, 4, 16, 4),
+          child: Row(children: [
+            // ClipRRect(
+            //   borderRadius: BorderRadius.circular(styleModel.fontSize),
+            //   child: Image(
+            //       image: NetworkImageWithRetry(model.profilePictureUrl),
+            //       height: styleModel.fontSize * 1.5),
+            // ),
+            const SizedBox(width: 12),
+            Expanded(
+              child: RichText(
+                text: TextSpan(
+                  children: [
+                    TextSpan(text: name, style: boldStyle),
```

Comment:
Is there a cheer emote chain associated with the message from twitch? Seems like it would be better to render those emotes.